### PR TITLE
PRO-6269: Update jQuery and improve Document Upload page

### PR DESCRIPTION
**Pull request by LucaDelBuonoHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRO-6269

### Change description ###
Update jQuery to the latest version and improve Document Upload page

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: app/steps/ui/documentupload/template.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
         <p class="govuk-body">{{ content.paragraph1 }}</p>
 
         {{ govukInsetText({
-            text: (content.help1 + "<br><br>" + content.help2) | safe
+            text: ("<p class=\"govuk-body\">" + content.help1 + "</p><p>" + content.help2 + "</p>") | safe
```
Comment (davejones74) [inline app/steps/ui/documentupload/template.html]: second paragraph missing govuk-body class?

File: app/views/includes/scripts.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<script src="/public/javascripts/jquery-1.11.3.js"></script>
+<script src="/public/javascripts/jquery/jquery.min.js"></script>
```
Comment (davejones74) [inline app/views/includes/scripts.html]: shouldn't the min file include the version, how do we know when to update in the future without breaking open the file.

Comment (LucaDelBuonoHMCTS) [reply app/views/includes/scripts.html]: No it doesn't :)

File: app/steps/ui/documentupload/template.html (hunk 1 above)
Comment (LucaDelBuonoHMCTS) [reply app/steps/ui/documentupload/template.html]: You're right, I'll add it.

**Pull request merged**